BaseFilter/sections/adservers.txt

```diff
@@ -9,6 +9,7 @@
 !
 !
 !
+||tolerancewarily.com^
```

Comment:
Maybe the site which loaded this ad also can be added to denyallow rule?
https://github.com/AdguardTeam/AdguardFilters/blob/d75c57d7a8718afd90c9c5f6ecc2eb3cdeef0e1b/BaseFilter/sections/specific.txt#L836